rclcpp/include/rclcpp/subscription_options.hpp

```diff
@@ -75,6 +75,70 @@ struct SubscriptionOptionsBase
   TopicStatisticsOptions topic_stats_options;
 
   QosOverridingOptions qos_overriding_options;
+
+  /// Enable use_default_callbacks setting.
+  SubscriptionOptionsBase &
+  enable_use_default_callbacks()
```

Comment:
maybe `set_use_default_callbacks(bool)` is a better idea.
@wjwwood @mabelzhang any preference?

Same comments apply elsewhere.

Reply:
Anyways, I would prefer `enable_default_callbacks` than `enable_use_default_callbacks`.

Reply:
Since this is a binary option using true/false, I think we should do `set_use_default_callbacks(bool)` as you suggested.

Same for `ignore_local_publications`.

Reply:
Thank you for comments.

I choose enable/disable because `use_intra_process_comm` is not bool.
But as in comments below, it looks OK to treat it as if this is bool because one-liners are the target.

In 5ffeb9b, I think we can unify `set_some(bool)`.